### ISSUE TITLE
[Question]Is there a way to calculate the Coefficient of determination(R^2)?

### ISSUE THREAD

**Issue by yumetodo** [DomiDre/funcfit#12]
I want to calculate the Coefficient of determination(R<sup>2</sup>). 
For example, Microsoft Excel can do it like below:
![image](https://user-images.githubusercontent.com/10869046/68074724-b0f7a780-fde1-11e9-80b9-1a9da03aa809.png)

Background:

Microsoft Excel can calculate R<sup>2</sup> but cannot fit data to `y = a + b(a=1)`.
This tool can fit data to `y = a + b(a=1)` like below but cannot find the way to get R<sup>2</sup>

![image](https://user-images.githubusercontent.com/10869046/68074770-409d5600-fde2-11e9-87f8-e089265a8d78.png)

**Comment by DomiDre** [comment]
Thanks for the suggestion! 
I followed the wikipedia article definition of R^2 and added it in the fit statistics output. The actual calculation is found in the Rust package rusfun (https://github.com/DomiDre/rusfun/blob/master/src/curve_fit.rs#L343-L352).

Additionally I added an export button to be able to download the shown data & parameters & model in a text file for storage / further evaluation. Just shout out if you have any idea for further convenience functions :-).
